cypress/component/ResultContainer.cy.tsx

```diff
@@ -0,0 +1,66 @@
+import ResultContainer from '../../src/components/ResultContainer';
+import { protectedResponse2 } from '../fixtures/mocked-responses';
+
+describe('ResultContainer', () => {
+  it('Displays a set of Result Cards, select all checkbox, disabled download result buttons, summary stats, and how to get data modal button', () => {
+    cy.mount(<ResultContainer result={protectedResponse2} />);
+    cy.get('[data-cy="card-http://neurobagel.org/vocab/cool-dataset"]').should('be.visible');
+    cy.get('[data-cy="card-http://neurobagel.org/vocab/not-so-cool-dataset"]').should('be.visible');
+    cy.get('[data-cy="select-all-checkbox"]').should('be.visible');
+    cy.get('[data-cy="summary-stats"]')
+      .should('be.visible')
+      .should('contain', 'Summary stats: 2 datasets, 4 subjects');
+    cy.get('[data-cy="participant-level-download-results-button"]')
+      .should('be.visible')
+      .should('be.disabled');
+    cy.get('[data-cy="dataset-level-download-results-button"]')
+      .should('be.visible')
+      .should('be.disabled');
+    cy.get('[data-cy="how-to-get-data-modal-button"]').should('be.visible');
+  });
+  it('Selecting a dataset should enable the download result buttons', () => {
+    cy.mount(<ResultContainer result={protectedResponse2} />);
+    cy.get('[data-cy="card-http://neurobagel.org/vocab/cool-dataset-checkbox"] input').check();
```

Comment:
this is minor, but as is, the test is more like a mini-integration test. To check if download button styles correctly if any dataset is selected, you could just feed the selectedDatasets prop non-empty. Here you're testing that step, but also the correct event handling for a checked dataset card, and then whether the ResultContainer processes this info correctly. So 3 steps in one. The goal would be to really just test one thing

Reply:
Same decision as below: let's revisit this in a later issue / PR and see if can think different thoughts about it.